docs/README.md

```diff
@@ -1,112 +1,132 @@
-# How proxless works
-  
-> Proxless looks for any kubernetes services with specific annotations and make their associated deployment serverless.
- 
-It works with 2 modes:
+# How works proxless
+
+## Wording
+
+- `service` == `kubernetes service`
+- `deployment` == `kubernetes deployment`
+
+## In 1 minute
+
+The main idea when building proxless is to be as simple as possible.  
+You don't wanna install a Custom Resource Definition, you don't want to have a big stack, you just want something to scale up and down your deployment based on-demand.  
+Proxless is a simple proxy written in golang and does not consume much resources. You don't need anything else.
+
+You just need to add an annotation on the services you want to make on-demand and Proxless will handle the rest.
+
+Start now and deploy the [helm chart](../helm/README.md).  
```

Comment:
comment - prefer to have "easy one link to install" here in main readme, but advance control -> link to helm.

Reply:
I will add both then yeah.
I am not a fan of the `kubectl apply` cuz it's not configurable but having both should be fine. Anyway u're right, the example needs to be super simple